utils/dashboard.js

```diff
@@ -0,0 +1,30 @@
+export function getDashboardURL(endpoint) {
+  return `${
+    process.env.NEXT_PUBLIC_DASHBOARD_API_URL ||
+    "http://dashboard.sandy-web.ad.renci.org/api/webinfo/"
+  }${endpoint}`;
+}
+
+// Helper to make GET requests to Dashboard
+export async function fetchFromDashboard(endpoint, options = {}) {
+  const defaultOptions = {
+    headers: {
+      accept: "application/json",
+    },
+  };
+  const mergedOptions = {
+    ...defaultOptions,
+    ...options,
+  };
+  const requestUrl = getDashboardURL(endpoint);
+  process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0;
```

Comment:
What was the reason for this? Is this only happening in dev? Node docs highly discourage its use 

From: https://nodejs.org/api/cli.html
> If value equals '0', certificate validation is disabled for TLS connections. This makes TLS, and HTTPS by extension, insecure. The use of this environment variable is strongly discouraged.

Reply:
Dashboard API is over http currently, we will remove this once that has a certificate.